app/views/account/comments/_show_comments.html.slim

```diff
@@ -1,16 +1,14 @@
-- @comments.each do |comment|
+- comments.each do |comment|
   article.media.box
     .media-content
       .content
         p
-          strong
-            = comment.user.full_name
+          = simple_format(comment.user.full_name, {}, wrapper_tag: "strong")
```

Comment:
USE this style https://bulma.io/documentation/layout/media-object/ 

BUT http://prntscr.com/ld1mgx without nesting